crates/cli/Cargo.toml

```
@@ -44,4 +47,11 @@ name = "reference"
 harness = false
 
 [features]
-vendored-openssl = ['openssl/vendored']
+default = ["rustls-tls"]
+
+native-tls = ["dep:native-tls", "ureq/native-tls"]
+rustls-tls = ["ureq/tls"]                          # ureq uses rustls by default.
+
+# Legacy support
+openssl = ["native-tls"]
```

```suggestion
openssl = ["dep:native-tls"]
```
We can have the `dep` here so we don't need it above, this is all legacy features anyway.